.flake8

```diff
@@ -0,0 +1,15 @@
+[flake8]
+
+# no lines over 120 allowed
+max-line-length = 120
+
+# exclude these dirs
+exclude = venv
+
+# show us where we got stuff wrong
+show_source = True
+
+# include our stuff
+select = E,F,W,C,I
+
+statistics = True
```

Comment:
I assume you took these from Zebo, @omesser was the one to add flake8 to MLRun and he did things differently:
* Exclude also have `.git` 
* Not specifying `show_source`, `select`, `statistics`
* Adding `extend-ignore = E203, W503`

I suggest to add get this lint addition out of this PR, and consult with him what was the reason for the differences

Reply:
I will take it offline with @omesser and will retain the flake8 config changes on this PR
a followup PR with some `black` and etc, would be later on, after adding github action to run lint / unit tests.